### ISSUE TITLE
Convoy container fails to start

### ISSUE THREAD

**Issue by sshipway** [rancher/convoy#53]
Rancher 0.47.0, convoy-agent 0.1.0, Docker 1.7.1, CoreOS 766.5.0

A glusterfs stack was successfully started up using the Catalogue.

Using the catalogue, attempted to start a Convoy stack over the top.  The storagepool service started, but the main convery service does not start; the container exits with the error message:
```
26/11/2015 2:50:48 pmWaiting for metadatatime="2015-11-26T01:50:48Z" level=fatal msg="json: cannot unmarshal string into Go value of type []*configs.ThrottleDevice"
```

Interestingly, the error message about 'cannot unmarshal' is similar to another error we are seeing in Rancher when loading docker-compose.yml.  Could this be down to an incompatibility with CoreOS?

**Comment by deniseschannon** [comment]
Closed in favor of https://github.com/rancher/rancher/issues/2863

**Comment by deniseschannon** [comment]
Quick Question: @sshipway what do you mean by 

"Interestingly, the error message about 'cannot unmarshal' is similar to another error we are seeing in Rancher when loading docker-compose.yml. "

**Comment by sshipway** [comment]
We had another problem with rancher that produced a "cannot unmarshal" error.  This turned out to be where a stack had been previously created under 0.44; when exported it included a parameter that was no longer valid under 0.47.  Back when I made the above comment I had thought it might be a coreos incompatibility but the error turned out to be a rancher backwards-compatibility issue.

TLDR - I no longer think the two problems are related
